src/main/java/uk/gov/justice/hmpps/prison/api/model/OffenderNonAssociation.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@ApiModel(description = "Offender non-association")
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@EqualsAndHashCode
+public class OffenderNonAssociation {
```

Comment:
I should probably add the offender name to this also???

Reply:
That would save us an extra call to get the offender details when we come to use this :)

Reply:
I should probably also add the location of the prisoners...

Reply:
Added first and last name.  Still need to add locations.